docs/developer-docs/latest/setup-deployment-guides/installation/digitalocean-one-click.md

```diff
@@ -5,166 +5,45 @@ description: Quickly deploy a Strapi application on DigitalOcean by simply using
-
-## Default server configuration
-
-From the initial startup you should not need to configure anything to get started, there is some included software that is configured:
-
-- Node.js v12 (installed via the offical [apt repository](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions))
-- Yarn Latest Stable (installed via the official [apt repository](https://classic.yarnpkg.com/en/docs/install/#debian-stable))
-- Nginx Latest (Ubuntu default repository)
-- UFW (Uncomplicated Firewall)
-  - Configured to only allow incoming ports: 80 (HTTP), 443 (HTTPS), and 22 (SSH)
-- PostgreSQL Latest (Ubuntu default repository)
-- PM2 (Installed globally using Yarn)
-
-## File and Software paths
-
-### Nginx
-
-The DigitalOcean one-click application uses Nginx to proxy http on port 80 to Strapi, this is to ensure the system is secure as running any application on ports below 1024 require root permissions.
-
-The example config included by default is located at `/etc/nginx/sites-available/strapi.conf` and the upstream block is located at `/etc/nginx/conf.d/upstream.conf`
-
-To learn more about the Nginx proxy options you can view the Nginx proxy [documentation](http://nginx.org/en/docs/http/ngx_http_proxy_module.html).
-
-:::: tabs
-
-::: tab strapi.conf
-Path: `/etc/nginx/sites-available/strapi.conf`
-
-```
-server {
-
-# Listen HTTP
-    listen 80;
-    server_name _;
-
-# Proxy Config
-    location / {
-        proxy_pass http://strapi;
-        proxy_http_version 1.1;
-        proxy_set_header X-Forwarded-Host $host;
-        proxy_set_header X-Forwarded-Server $host;
-        proxy_set_header X-Real-IP $remote_addr;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header X-Forwarded-Proto $scheme;
-        proxy_set_header Host $http_host;
-        proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "Upgrade";
-        proxy_pass_request_headers on;
-    }
-}
-```
 
+:::warning PREREQUISITES
+A DigitalOcean account is necessary to follow this installation guide. Please visit the [DigitalOcean website](https://try.digitalocean.com/strapi/) to create an account if you don't already have one.
 :::
 
-::: tab upstream.conf
-Path: `/etc/nginx/conf.d/upstream.conf`
-
-```
-upstream strapi {
-    server 127.0.0.1:1337;
-}
-```
-
-:::
+## Creating a Strapi project
 
-::::
+1. Go to the [Strapi page on DigitalOcean's marketplace](https://marketplace.digitalocean.com/apps/strapi).
+2. Click on **Create Strapi Droplet** button.
+3. Keep the selected "Shared CPU - Basic" plan.
+4. Choose "Regular Intel with SSD" as a CPU option.
+5. Select your virtual machine size (minimum of 2 GB/1 CPU).
+6. Choose a datacenter region (closest to you or your target area).
+7. Add a new SSH key. If you are on Windows you can follow [this guide](https://www.digitalocean.com/docs/droplets/how-to/add-ssh-keys/create-with-putty/).
```

Comment:
Because on Unix-based systems (Linux & macOS) the tools are already built-in, while you need to use a 3rd party tool on Windows.

From [DigitalOcean docs](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/):
> Create an SSH key using an SSH client installed on your local computer. OpenSSH is included on Linux, macOS, and Windows Subsystem for Linux. Windows users without Bash can use PuTTY.

At least I guess this is the idea that was behind @derrickmehaffy 's initial sentences.

Reply:
What do you think about this suggestion?
(Replacing by a more general link that covers both methods, on macOS/Linux and on Windows, still in DigitalOcean docs)

```suggestion
7. Add a new SSH key. You can follow [this guide](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/).
```

Reply:
Lgtm, I wrote this guide before windows included most of the ssh tooling in powershell.